p2p/host/basic/basic_host.go

```diff
@@ -256,6 +257,11 @@ func (h *BasicHost) newStreamHandler(s inet.Stream) {
 	go handle(protoID, s)
 }
 
+// PushIdentify pushes an identify update through the identify push protocol
+func (h *BasicHost) PushIdentify() {
```

Comment:
Let's add a comment saying that this is an unstable interface. Ideally, we'd have an address manager, a protocol muxer, etc. that all emit events. Then, the identify service would subscribe to these events to determine when and if to push information to peers.

Reply:
Sure, will add a comment.